pkg/apis/core/types_shoot.go

```diff
@@ -723,6 +723,13 @@ const (
 	CRINameContainerD CRIName = "containerd"
 )
 
+// ContainerRuntimeName is a type alias for the ContainerRuntime name string.
+type ContainerRuntimeName string
+
+const (
+	ContainerRuntimeGVisor ContainerRuntimeName = "gvisor"
```

Comment:
Why is this here?
I don't think, there should be any known provider names in Gardener's API...
Btw, isn't this a clear case for an integration test, that should be part of the respective Extension repo?